internal/vpn/os_server_linux.go

```diff
@@ -139,20 +150,24 @@ func EnableIPMasquerading(ifcName string) error {
 // DisableIPMasquerading disables IP masquerading for the interface with name `ifcName`.
 func DisableIPMasquerading(ifcName string) error {
 	cmd := fmt.Sprintf(disableIPMasqueradingCMDFmt, ifcName)
-	//nolint:gosec
-	if err := exec.Command("sh", "-c", cmd).Run(); err != nil {
+	if err := osutil.Run("sh", "-c", cmd); err != nil {
 		return fmt.Errorf("error running command %s: %w", cmd, err)
 	}
 
 	return nil
 }
 
 func getIPForwardingValue(cmd string) (string, error) {
-	outBytes, err := exec.Command("sh", "-c", cmd).Output() //nolint:gosec
+	out, err := osutil.RunWithResult("sh", "-c", cmd)
 	if err != nil {
 		return "", fmt.Errorf("error running command %s: %w", cmd, err)
 	}
 
+	outBytes, err := ioutil.ReadAll(out)
+	if err != nil {
+		return "", fmt.Errorf("failed to read stdout: %w", err)
+	}
+
```

Comment:
Could []byte be returned directly, or we need io.Reader somewhere? It seems to me this reading is repeated quite a bit, plus error reporting could be put inside osutil as well and make it report the name of the command.

Reply:
@i-hate-nicknames well spotted, I thought of it. and probably yeah, it would be good. but I'll make a separate func for it, leaving this one with `io.Reader` available just in case it's needed somewhere